### PR TITLE
Adding value to where in toolGetMapping

### PR DESCRIPTION
**Pull request by whitehacker**
Went through all warnings displayed by `reposearch toolGetMapping` in `mrremind` and adjusted the `where` value.

### CODE REVIEW

**Comment by LaviniaBaumstark** [comment]
Did you try that input data generation is still working? Did you compare old and new input data and ensured that they are the same?

File: R/calcAGEB.R [hunk 1]
```diff
@@ -15,7 +15,7 @@
 calcAGEB <- function(subtype = "balances") {
   ageb <- readSource("AGEB", subtype = subtype)
 
-  mapping <- toolGetMapping("Mapping_AGEB_REMIND.csv", type = "reportingVariables") %>%
+  mapping <- toolGetMapping("Mapping_AGEB_REMIND.csv", type = "reportingVariables", where="mappingfolder") %>%
```
Comment (fbenke-pik) [inline R/calcAGEB.R:18]: Please make sure that your added code does not produce linter warnings. This addition should cause a complaint about missing whitespaces.

**Review by fbenke-pik — CHANGES_REQUESTED** [review]

File: R/calcAGEB.R (hunk 1 above)
Comment (whitehacker) [reply R/calcAGEB.R]: [Added](https://github.com/pik-piam/mrremind/pull/405/commits/16631c6857f80c78cf27909126dbe3c8a8407076).

**Comment by whitehacker** [comment]
> Did you try that input data generation is still working? Did you compare old and new input data and ensured that they are the same?

Yes, 
New Data: REMIND: `/p/projects/rd3mod/inputdata/output/rev6.54baseerInputData_2b1450bc_remind.tgz`
                  MAgPIE: `/p/projects/rd3mod/inputdata/output/rev4.87baseerMP_h12_fd712c0b_cellularmagpie_c200_MRI-ESM2-0-ssp370_lpjml-8e6c5eb1.tgz`
                  
Comparison: inputdata-comparedata /p/projects/rd3mod/inputdata/output/rev6.54baseerInputData_2b1450bc_remind.tgz /p/projects/rd3mod/inputdata/output/rev6.54falks-cache_2b1450bc_remind.tgz

**Comment by LaviniaBaumstark** [comment]
did you check that you are not generating linter warnings?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
```
> invisible(system('git fetch --multiple origin baseer-github', intern = TRUE))
> invisible(system('git checkout -f baseer-github/master', intern = TRUE))
HEAD is now at dc7c6dd Fixed the error in R/calcEmissionFactors.R
> 
> suppressMessages(lint_new <- lintr::lint_package(linters = lucode2::lintrRules()))
> 
> invisible(system('git checkout -f origin/master', intern = TRUE))
Previous HEAD position was dc7c6dd Fixed the error in R/calcEmissionFactors.R
HEAD is now at 00c7fb9 Merge pull request #404 from orichters/NDC2023
> 
> suppressMessages(lint_old <- lintr::lint_package(linters = lucode2::lintrRules()))
> 
> c(length(lint_old), length(lint_new), length(lint_new) - length(lint_old))
[1] 23451 23492    41
```
A 0.2 % increase.

**Review by LaviniaBaumstark — APPROVED** [review]

**Review by fbenke-pik — APPROVED** [review]

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Not what I would have expected under a "fixed lintr warnings" commit, and somewhat obfuscating the commit history.
``` diff
$ git diff --ignore-all-space -U0 c09aea9~ -- R/read*
diff --git a/R/readExpertGuess.R b/R/readExpertGuess.R
index 04b18a4..9057ff8 100644
--- a/R/readExpertGuess.R
+++ b/R/readExpertGuess.R
@@ -40 +40 @@ readExpertGuess <- function(subtype) {
-    a <- read.csv("co2prices.csv", sep = ";")
+    a <- read.csv("co2prices-2023-06.csv", sep = ";")
diff --git a/R/readUNFCCC_NDC.R b/R/readUNFCCC_NDC.R
index b9fa3ed..9a2850c 100644
--- a/R/readUNFCCC_NDC.R
+++ b/R/readUNFCCC_NDC.R
@@ -21 +21 @@ readUNFCCC_NDC <- function(subtype) {
-    NDCfile <- "NDC_2023-02-24.xlsx"
+    NDCfile <- "NDC_2023-05-31.xlsx"
```
Would have appreciated a `rebase` here.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
They very much are, because `c09aea9` does not just "fix lintr warnings" but also merges upstream changes.  Like I said, obfuscating what is going on.
```
$ git log --graph --pretty='oneline' --abbrev-commit 42e7190..HEAD
*   623de95 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #405 from whitehacker/master
|\  
| * 7e18f48 Fixed lintr warnings
| *   c09aea9 Fixed lintr warnings
| |\  
| |/  
|/|   
* |   2eed46a Merge pull request #406 from orichters/co2prices
|\ \  
| * | 16b1597 buildLibrary
| * | 3da7cfc in mrremind/R/calcCO2Prices.R add to description that 2020 CO2 prices are added
|/ /  
* |   00c7fb9 Merge pull request #404 from orichters/NDC2023
|\ \  
| * | 7021b1c update NDC targets to 2023-05-31
|/ /  
| * 4b7da3d Fixed lintr warnings
| * dc7c6dd (baseer-github/master) Fixed the error in R/calcEmissionFactors.R
| * 16631c6 spacing out the equal sign in where = mappingfolder
| * fc72983 (fix/all_hail_the_lintr_god) Added v(mappingfolder)alue to w )re in toolGetMapping
| *   5789989 Merge branch 'master' of github.com:pik-piam/mrremind
| |\  
| |/  
|/|   
* |   fc8e2e8 Merge pull request #403 from mellamoSimon/calcEmissionFactorsFeedstocks
|\ \  
| * | 130b433 update version number
| * | e164907 bugfix emi factors for chem sector process emissions (zeros in USA)
|/ /  
* | 0136254 Merge pull request #401 from fbenke-pik/unfccc
* | 1be9ad1 increment version
* | 9cf3039 support reading in UNFCCC variables that cannot be referenced by fixed position
* | d1e248f remove UNFCCC variables that cannot be accurately parsed with the current logic
* | 695c099 Merge branch 'master' into unfccc
* | 3ca16fa (falk-personal/unfccc) Merge remote-tracking branch 'upstream/master' into unfccc
* | 8575f73 read UNFCCC data from 2023
 /  
* 11b64db Added value to whe(mappingfolder)re in toolGetMapping
```

**Comment by fbenke-pik** [comment]
Ok, I see. Not good. 
@whitehacker What happened [here](c09aea9)?
@0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q  Is it worth trying to fix this? Best without rewriting history?  If so, do you already have an idea how, or should I look into this?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> Ok, I see. Not good.

But not terrible, either.  Just inconvenient.

> Is it worth trying to fix this? 

No.  

> Best without rewriting history?

You cannot change the history of a public repository without calling down the ire of the git gods upon yourself.

**Pull request merged**